docs/introduction/introduction.md

```diff
@@ -0,0 +1,80 @@
+## Visual DL
+`Visual DL`：让你的深度学习任务变得生动形象，可视化起来。
+
+目前大部分深度学习框架都是基于Python语言，训练过程的状态通过日志的方式记录下来，类似下面这种：
+
+```shell
+loss:[2.532566] acc:[0.0859375] pass_acc:[0.0859375]
+loss:[2.6916795] acc:[0.09375] pass_acc:[0.08984375]
+loss:[2.7038598] acc:[0.1171875] pass_acc:[0.09895834]
+loss:[2.6405232] acc:[0.125] pass_acc:[0.10546875]
+```
+
+这种方式可以短期看一下训练状态，但是很难从全局把握训练过程的一些变化趋势，提取的信息有限。
+
+Visual DL 让你方便的将训练过程可视化起来，形成类似下面的图片：
+<p align="center">
+<img src="scalar.png"/>
```

Comment:
scalar need 0.8 smooth, or it looks too steep.

Reply:
done